lib/fizzy/parser.hpp

```diff
@@ -26,6 +26,12 @@ struct parser
 {
 };
 
+template <>
+struct parser<uint32_t>
```

Comment:
Why is this needed?

Reply:
parser of vector requires parser of an element